.github/workflows/test.yml

```diff
@@ -26,19 +26,51 @@ jobs:
               exit 1
           fi
 
-      - name: make build
+      - name: make build-arm64
         run: |
-          sudo -u testbot bash -lc 'make build'
+          sudo -u testbot bash -lc 'make build-arm64'
 
+  test_32_bit: 
+      runs-on: buildjet-4vcpu-ubuntu-2204-arm
+      container: ghcr.io/viam-modules/raspberry-pi:arm32
+      timeout-minutes: 30
+  
+      steps:
+        - uses: actions/checkout@v3
+  
+        - name: Verify no uncommitted changes from make lint
+          run: |
+            git init
+            git add .
+            chown -R testbot:testbot .
+            sudo -u testbot bash -lc 'make lint'
+            if [ -n "$GEN_DIFF" ]; then
+                echo '"make lint" resulted in changes not in git' 1>&2
+                git status
+                exit 1
+  
+        - name: Build for ARM32
+          run: |
+            sudo -u testbot bash -lc 'make build-arm32'
+  
```

Comment:
Instead of duplicating, you should do a matrix. See RDK's process here: https://github.com/viamrobotics/rdk/blob/main/.github/workflows/test.yml#L19-L35